sasmodels/models/cylinder.py

```diff
@@ -45,7 +45,7 @@
 
 .. math::
 
-    P(q) = \frac{\text{scale}}{V}
+    I(q) = \frac{\text{scale}}{V}
         \int_0^{\pi/2} F^2(q,\alpha) \sin \alpha\ d\alpha + \text{background}
 
```

Comment:
You may want to move the u-substitution comment down. That is,

P(q) = F²(q) =∫₀^π/2 F²(q, α) sin α dα

and so

I(q) = scale/V ∫₀^π/2 F²(q, α) sin α dα + background

After substitution with u = cos α, du = sin α dα,

P(q) = ∫₀¹ F²(q, u) du
I(q) = scale/V ∫₀¹ F²(q, u) du + background